releases/release-1.31/release-notes/maps/pr-125437-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 125437
```

Comment:
We might want to clarify that this deprecates the kubectl exec command execution without dash: https://github.com/kubernetes/kubernetes/pull/125437